G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/OutlookSaveAttachment/ShouldSaveAttachment_WhenSaveAttachmentCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,27 @@
+﻿♥outlookInboxFolder = ‴t.g1ant.test@gmail.com\Skrzynka odbiorcza‴
+♥toBeSentAttachmentPath = ‴♥environment⟦TEMP⟧\testattachmentsent.txt‴
+♥toBeDownloadedAttachmentPath = ‴♥environment⟦TEMP⟧\testattachmentdownloaded.txt‴
+
+outlook.open display false
+text.write text ‴Test text.‴ filename ♥toBeSentAttachmentPath
+
+♥attachmentsToBeSent = ⟦list⟧♥toBeSentAttachmentPath❚ 
+
+outlook.newmessage to ‴t.g1ant.test@gmail.com‴ subject ‴Test mail‴ body ‴Test content.‴ attachments ♥attachmentsToBeSent
+outlook.send 
+delay 5
+
+outlook.open display false
+delay 5
+outlook.getfolder path ♥outlookInboxFolder result ♥InboxFolder
+♥emails = ♥InboxFolder⟦mails⟧
+♥mail = ♥emails⟦♥emails.Count⟧
+♥attachments = ♥mail⟦attachments⟧
+♥attachment = ♥attachments⟦1⟧
+outlook.saveattachment ♥attachment path ♥toBeDownloadedAttachmentPath
+text.read filename ♥toBeDownloadedAttachmentPath result ♥textFromDownloadedAttachemnt
+test ⊂♥textFromDownloadedAttachemnt.Equals("Test text.")⊃
+outlook.close
+
+file.delete ♥toBeSentAttachmentPath
```

Comment:
In other tests, you are firstly check if the file exists. Why aren't you doing it here? If the file.delete checks if the file exists before then the checking in other tests does not make sense, and if it does not (and throws exception) then it is needed here as well.

Reply:
ping

Reply:
done